docgen/parser/parser.go

```diff
@@ -27,11 +27,15 @@ const (
 )
 
 // ParseFile parses a proto file.
-func ParseFile(file *google_protobuf.FileDescriptorProto) (*File, error) {
+func ParseFile(file *FileDescWrapper) (*File, error) {
 	comments := parseComments(file.GetSourceCodeInfo())
 
 	pkgName := file.GetPackage()
-	messages := parseMessages(pkgName, comments, file.GetMessageType())
+	//nested,err := nestedDescriptorMessages(file, dependencies)
```

Comment:
remove this too. please try to not keep commented out code in PRs, unless there's a good reason for it

Reply:
Oh, i haven't cleaned it up yet and removed the comments. Would do so now. Just wanted to make sure it worked fo @favadi

Reply:
Well, you did clearly ask for a review+merge ;)

Reply:
Sorry, wrong PR!